libraries/abstractions/pkcs11/mbedtls/iot_pkcs11_mbedtls.c

```diff
@@ -3836,6 +3813,7 @@ CK_DECLARE_FUNCTION( CK_RV, C_GenerateRandom )( CK_SESSION_HANDLE xSession,
 {
     CK_RV xResult = CKR_OK;
     int lMbedResult = 0;
```

Comment:
We're not calling mbed routines here anymore so this name becomes confusing. However, inconveniently, CK_RV is unsigned while the Port call below returns signed. Still, since we're not capturing a specific status code from the Port call, it seems that we don't need a separate local variable.

Reply:
Added a PKCS print on error, so I will rename to be lResult so I can still print that out.